package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "discord-anti-spam",
-  "version": "2.4.1",
+  "version": "2.5.0",
```

```suggestion
  "version": "2.4.1",
```
Version should only be bumped on merge